File: src/ExamRegister.WebApi.Abstractions/DTO/MedicoDTO.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ExamRegister.WebApi.Abstractions.DTO
{
    public partial class MedicoDTO
    {
        public Guid idmedico { get; set; }
        public string nome { get; set; }
        public string email { get; set; }
        public string celular { get; set; }
        public string telefone { get; set; }
        public string telefone2 { get; set; }
        public string crm { get; set; }
        public DateTime? inativo { get; set; }

        public IEnumerable<MedicoClinicaDTO> medicoClinica { get; set; }
    }
}

File: src/ExamRegister.UI/src/app/shared/DTOs/medico-dto.ts
import { MedicoclinicaDTO } from "./medicoclinica-dto";

export interface MedicoDTO {
    idmedico: string;
    nome: string;
    email: string;
    celular: string;
    telefone: string;
    telefone2: string;
    crm: string;
    inativo?: Date;

    medicoClinica: MedicoclinicaDTO[];
}
